MREUnityRuntime/MREUnityRuntimeLib/link.xml

```diff
@@ -0,0 +1,6 @@
+<linker>
+  <assembly fullname="MREUnityRuntimeLib" preserve="all"/>
+  <assembly fullname="System">
+    <type fullname="Mono.Net.Security.LegacyTlsProvider" preserve="all" />
+  </assembly>
+</linker>
```

Comment:
Is this all? I thought the MRE runtime required System.Net included as well.

Reply:
This is copied directly from the Test Bed link.xml file.  I would have thought this would be all inclusive of what we need to run.

Reply:
I also do not see the one you mention in the Altspace link.xml file either.

Reply:
I confused csc.rsp with link.xml. This is only needed by the compiler, not the linker. Approved.